src/css/story/year-slider.styl

```diff
@@ -1,21 +1,21 @@
 // Height of SVG in terms of viewBox/user units
-$height = 50px
+$height = 70px
```

Comment:
Anecdotally, I feel like the years were fitting into the available space better before these changes. On mobile (emulator) I'm seeing some clipping which I don't think was there before. On desktop, I'm seeing the years hovering some distance from the line (whereas before I think they were flush with it). I don't know if that was a deliberate decision, but I personally prefer the aesthetics of the previous layout.

(But props for managing to work with this part of the CSS at all. I left you quite a mess.)

Reply:
Was just trying to give all of this a bit more breathing room. I'm planning on sizing down the radar a bit on desktop so that everything's just a bit more open. It all felt really cramped up there to me.